### DISCUSSION TITLE
Uninitialized string offset: 0 in .../Mpdf.php on line 16600

### DISCUSSION THREAD

**Discussion by INewto** [mpdf/mpdf#1503] [Q&A] [answered]
$mpdf->WriteHTML($css, 1); // this code making subj notice
$mpdf->WriteHTML($html, 2); // it's ok

$css -- some css code (checked, all ok)

So I can't to join my styles to pdf. What is this and how to fix?

**Comment by finwe** [comment, marked as answer]
Noone can help you unless you show your code and share details of your stack, especially the version of mPDF you are using.

**Reply by INewto** [reply]
Actually you already did, thank you :)
Problem solved by updating mpdf version.
But... I got another, that mpdf not supporting flexbox :/

**Reply by finwe** [reply]
https://github.com/mpdf/mpdf/issues/399#issuecomment-509927758
